dhcpv4/dhcpv4.go

```diff
@@ -682,6 +682,20 @@ func (d *DHCPv4) ValidateOptions() {
 	}
 }
 
+// IsOptionRequested function takes a DHCPv4 message and an OptionCode, and
```

Comment:
the comment should be slightly updated - DHCPv4 is not anymore an input parameter, so just say "IsOptionRequested takes an OptionCode" etc, or even simpler, "IsOptionRequested returns whether the provided option code is found in the requested options of the DHCPv4 message"

Reply:
Right. I am going to update it.